.github/workflows/build-staging-and-deploy-aws.yml

```diff
@@ -27,15 +27,6 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Set up Depot CLI
         uses: depot/setup-action@v1
-      - name: 📦 Build backend and export to Docker
```

Comment:
While this step isn't necessary and can just be removed, one note on caching: 

You were previously seeing this image rebuild even though it seemed it it should have been cached - that was because the value of `INFISICAL_PLATFORM_VERSION` was different (it's not set here, and it is set below), which caused a re-run of the Next.js build. It would be possible to make the cache work by defining the same `INFISICAL_PLATFORM_VERSION` build arg here, if you did need to keep this step for some reason.

Reply:
Wow i didn't even see we had a duplicate